src/list/ListItemAccordion.tsx

```diff
@@ -10,10 +10,9 @@ export type ListItemAccordionProps = ListItemProps & {
   icon?: IconNode;
   expandIcon?: IconNode;
   content?: React.ReactNode;
-  noAnimation?: boolean;
   noRotation?: boolean;
   noIcon?: boolean;
-  animationDuration?: number;
```

Comment:
IMO `animation` should be an object | boolean , with various properties one of which can be `duration`.

Reply:
Sure Sir